src/services/API_carModel.ts

```diff
@@ -33,24 +35,11 @@ export const fetchAllCarData = async () => {
     }
 }
 
-export const fetchCarImageByColorAndVariant = async (modelName: string, color: string, wheelVariant: number, position: CarPosition) => {
+export const fetchCarImageByColorAndVariant = async (modelName: "Audi RS5" | "Audi RS6" | "Audi e-tron GT", color: string, wheelVariant: number, position: CarPosition) => {
```

Comment:
When you have more than 2-3 props on a function, it's better to pass them as an object, especially when some of them are optional.
```
interface Props {
  modelName: ...
  ...
}

export const fetchCarImageByColorAndVariant = async ({modelName, ...}: Props) => ...
```

Reply:
Created interfaces for functions with multiple props and passed the props as objects.